spec/OBD/OBD.vspec

```diff
@@ -43,6 +44,11 @@
   enum: [ "spark", "compression" ]
   description: Type of the ignition for ICE - spark = spark plug ignition, compression = self-igniting (Diesel engines)
 
+- DTCList:
+  datatype: string[]
+  type: sensor
+  description: List of cuurently active DTCs formatted according OBD II standard ([P|C|B|U]XXXXX )
```

Comment:
*currently

Reply:
Is there a more formal way to specify the document you refer to when you say "OBD II standard"?

Reply:
Added reference to SAE standard, see current commit